package.json

```diff
@@ -123,10 +124,8 @@
     "stylelint-selector-bem-pattern": "^2.1.1",
     "svg-to-ts": "^8.8.1",
     "svgo": "^2.7.0",
-    "ts-mockito": "^2.6.1",
     "ts-node": "^10.9.1",
-    "tsickle": "^0.46.3",
-    "typescript": "4.3.5",
+    "typescript": "4.6.4",
```

Comment:
How about `4.7.4`? Not critical though...

Reply:
Not sure why it was updated to that version - I've now updated to the latest `4.8.2` which seem to be supported by `14`